File: Python/Algorithm/bubble_sort.py
def bubble_sort(li: list[int]) -> list[int]:
    for i in range(0, len(li)):
        for j in range(i + 1, len(li)):
            if li[i] > li[j]:
                temp = li[i]
                li[i] = li[j]
                li[j] = temp

    return li


li = [5, 4, 3, 2, 1]
sorted_li = bubble_sort(li)

print(sorted_li)

File: Lua/Algorithms/bubbleSort.lua
function bubbleSort(array)
    for i = 1, #array, 1 do
        for j = i + 1, #array, 1 do
            if array[i] > array[j] then
                local temp = array[i]
                array[i] = array[j]
                array[j] = temp
            end
        end
    end
    return array
end


print(table.concat(bubbleSort({5, 4, 3, 2, 1}), ", "))
